scripts/package.sh

```diff
@@ -16,6 +16,8 @@ function main() {
       GOOS="${os}" GOARCH="amd64" go build -o "${ARTIFACTS_DIR}/jam-${os}" ./cargo/jam/main.go
       chmod +x "${ARTIFACTS_DIR}/jam-${os}"
     done
+    echo "* building jam on windows"
+    GOOS="windows" go build -o "${ARTIFACTS_DIR}/jam-windows.exe" ./cargo/jam/main.go
```

Comment:
Chose not to roll this into the for loop due to the file extension. This is a quick, simple solution, but happy to clean it up if needed :)